.github/workflows/commit_check.yaml

```
@@ -3,65 +1,67 @@
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements/dev.txt
 
-        - run: isort . --diff --check-only
-        - run: black --check --diff .
+  linter:
+    needs: setup-virtualenv
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)